src/www/status_ntpd.php

```diff
@@ -189,11 +189,11 @@
 								<tbody>
 								<?php if (isset($config['ntpd']['noquery'])): ?>
 								<tr><td class="listlr" colspan="11" align="center">
-									Statistics unavailable because ntpq and ntpdc queries are disabled in the <a href="services_ntpd.php">NTP service settings</a>.
+									<?=gettext('Statistics unavailable because ntpq and ntpdc queries are disabled in the') ?> <a href="services_ntpd.php"><?=gettext('NTP service settings') ?></a>.
```

Comment:
Need this html wrapped in sprintf like so: https://github.com/opnsense/core/blob/6612086c2135ce8b13a93915822694d2a78b5225/src/www/diag_sockets.php#L53